tests/integration/test_slide.py

```diff
@@ -24,6 +25,36 @@ def it_knows_its_name(self):
 
         assert name == ntpath.basename(SVS.CMU_1_SMALL_REGION).split(".")[0]
 
+    @pytest.mark.parametrize(
+        "use_largeimage, slide_props",
+        [
+            (True, None),
+            (False, None),
+            (False, {"aperio.MPP": 0.499}),
+            (
+                False,
+                {
+                    "tiff.XResolution": 20040.080160320642,
+                    "tiff.ResolutionUnit": "centimeter",
+                },
+            ),
+        ],
+    )
+    def it_knows_its_base_mpp(self, use_largeimage, slide_props):
+        slide = Slide(
+            SVS.CMU_1_SMALL_REGION,
+            os.path.join(SVS.CMU_1_SMALL_REGION, "processed"),
+            use_largeimage=use_largeimage,
+        )
+        if slide_props:
```

Comment:
instead of having an if here, is it possible to parametrize the slide properties?